_data/stage3.yml

```diff
@@ -225,6 +254,15 @@
   champions:
     - Jordan Harband
     - Kevin Gibbons
+  example: >
+    const entries = new Map([
```

Comment:
i think it's a bit disingenuous to call a Map `entries`; it's certainly *iterable* for entries, but that's not exactly the same thing. maybe call it `iterableOfEntries`?

Reply:
Or, should we use `[['cat', 'dog'], ['life', 42]]`?

Reply:
That could work too - but it's also nice to show that you can do it with a Map. Maybe both examples?

Reply:
Will just rename it to `iterableOfEntries`, too many examples could be confusing?

Reply:
Sounds good.